server/src/main/java/org/elasticsearch/index/engine/ReadOnlyEngine.java

```diff
@@ -583,4 +583,8 @@ public String getSearcherId() {
             }
         };
     }
+
+    public final String getCommitId() {
+        return commitId;
```

Comment:
NIT: Maybe make the field private now to avoid this happening elsewhere down the line?